test/e2e/suite/command/version_darwin.go

```diff
@@ -0,0 +1,31 @@
+//go:build darwin
+
+/*
+Copyright The ORAS Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package command
+
+import (
+	. "github.com/onsi/ginkgo/v2"
+	. "oras.land/oras/test/e2e/internal/utils"
+)
+
+var _ = Describe("ORAS darwin user:", func() {
```

Comment:
Looks like our CI build only runs on Linux.

Reply:
ORAS E2E can be run on Darwin but E2E backend registries cannot be started on Darwin.

This test can be run successfully on Darwin though since `oras version` doesn't interact with registry.